src/component/Proposals/Statistic/Chart.js

```diff
@@ -0,0 +1,120 @@
+import {
+    Chart as ChartJS,
+    CategoryScale,
+    LinearScale,
+    PointElement,
+    LineElement,
+    Title,
+    Tooltip,
+    Legend,
+} from "chart.js";
+import { Line } from "react-chartjs-2";
+import moment from "moment";
```

Comment:
Moment est deprecated comme library. I was just wondering if we should use another one? 


![image](https://user-images.githubusercontent.com/21199130/156810011-9e63523f-a188-410d-98b2-c0cb083b7116.png)

Reply:
Example new project: https://github.com/moment/luxon/